app/src/main/java/com/simply/birthdayapp/presentation/ui/screens/auth/register/RegisterScreen.kt

```diff
@@ -0,0 +1,35 @@
+package com.simply.birthdayapp.presentation.ui.screens.auth.register
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+
+import androidx.compose.material3.Text
```

Comment:
Format the code. We should not have blank lines here.

Reply:
ok